src/Unleash/Internal/UnleashServices.cs

```diff
@@ -17,6 +17,7 @@ internal class UnleashServices : IDisposable
         private static readonly ILog Logger = LogProvider.GetLogger(typeof(UnleashServices));
         private readonly CancellationTokenSource cancellationTokenSource = new CancellationTokenSource();
         private readonly IUnleashScheduledTaskManager scheduledTaskManager;
+        private readonly string connectionId = Guid.NewGuid().ToString();
```

Comment:
This will initiate a new connectionId per instance of the SDK which I assume is the intended behavior, ie if you create two SDK instances in the same app, these two instances will have different connection ids.

Reply:
that's the point. each new Unleash should get unique id so that we can find wrongly scoped SDKs

Reply:
If there are 2 different "polling instances", it's correct

Reply:
Great! Looks good!